Makefile

```diff
@@ -24,17 +24,17 @@ GENERATE_VAPBINDING ?= false
 
 VERSION := v3.21.0-beta.0
 
-KIND_VERSION ?= 0.27.0
+KIND_VERSION ?= 0.29.0
 KIND_CLUSTER_FILE ?= ""
 # note: k8s version pinned since KIND image availability lags k8s releases
-KUBERNETES_VERSION ?= 1.30.0
+KUBERNETES_VERSION ?= 1.33.0
 KUSTOMIZE_VERSION ?= 3.8.9
-BATS_VERSION ?= 1.8.2
-ORAS_VERSION ?= 0.16.0
+BATS_VERSION ?= 1.12.0
+ORAS_VERSION ?= 1.2.3
 BATS_TESTS_FILE ?= test/bats/test.bats
-HELM_VERSION ?= 3.7.2
-NODE_VERSION ?= 16-bullseye-slim
-YQ_VERSION ?= 4.30.6
+HELM_VERSION ?= 3.17.4
+NODE_VERSION ?= 24-bullseye-slim
```

Comment:
can you bump node version in the website generation workflow to make sure it works with 24

Reply:
otherwise, this'll break during release since this gets called when we version docs iirc

Reply:
Tested website generation make rule and its working as expected - https://github.com/JaydipGabani/gatekeeper/pull/131

Reply:
updated version to 24